install/kubernetes/helm/istio/values.yaml

```diff
@@ -81,6 +81,8 @@ tracing:
 #
 kiali:
   enabled: false
+  dashboard: 
```

Comment:
this should go in kiali values, not top level values.yaml. And also have a comment above as this becomes docs on istio.io